install-dev/data/xml/tab.xml

```diff
@@ -439,6 +439,9 @@
                 <tab id="Multistore" id_parent="Advanced_Parameters" active="0" enabled="1" hide_host_mode="0" icon="">
                     <class_name>AdminShopUrl</class_name>
                 </tab>
+                <tab id="ExperimentalFeature" id_parent="Advanced_Parameters" active="1" enabled="1" hide_host_mode="0" icon="">
+                  <class_name>AdminExperimentalFeature</class_name>
```

Comment:
Wouldn't it make sense to use `FeatureFlags` everywhere instead of mixing it with `ExperimentalFeatures`?

Reply:
BO page is named "Experimental Features".
Data model is named "Feature Flag".

I tried to make everything related to BO page with 1st name, and make everything else with 2nd name.

You suggest everything should be named Feature Flags, excluding the page title?

Reply:
Yep, best to use a simpler terminology. It can be labelled in any way later using translations.

Reply:
Fixed in https://github.com/PrestaShop/PrestaShop/pull/23703/commits/b8459900f4df0c8803d43ff6bccecaa3439df55e